Add db client connection limit (#1138)
pgbouncer/pgbouncer: doc/usage.md

current_connections

> :   Current number of connections for this database.

Reviewer: ```suggestion
:   Current number of server connections for this database.
```

Reviewer: and let's make the same change for `max_connections` above (I can't comment on that line because it's outside the diff according to github...)

Author: Done

max_client_connections

> :   Maximum number of allowed connections for this pgbouncer instance, as set by max_db_client_connections per database.

Reviewer: ```suggestion
:   Maximum number of allowed client connections for this pgbouncer instance, as set by max_db_client_connections per database.
```

current_client_connections

> :   Current number of connections for this pgboucer instance.

Reviewer: ```suggestion
:   Current number of client connections for this database.
```
